libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.h

```diff
@@ -380,6 +380,10 @@ typedef void (^SFOAuthBrowserFlowCallbackBlock)(BOOL);
  */
 @property (nonatomic, copy) NSString *brandLoginPath;
 
+/** Setup the coordinator to use SafariViewController for authenatication.
+ */
+@property (nonatomic, assign) BOOL requireCertificateAuthentication;
```

Comment:
Should we call that requireBrowserAuthentication?

Reply:
+1. Also, there's a typo in the comment: <i>authentication</i>.

Reply:
Also, is there a way to assign this property statically from the app, such that the app would always use browser auth? We probably want to avoid that.

Reply:
@bhariharan You can do so using SFUserAccountManager. @wmathurin We can call it requireBrowserAuthentication.

Reply:
@bhariharan I misunderstood, when you mean static are implying using the bootconfig plist or apps plist?